packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -142,7 +142,7 @@ module.exports = async (config = {}) => {
   })
 
   // Add plugins from the site config.
-  if (config.plugins) {
+  if (config.plugins) { // ???
```

Comment:
Again not sure if its enough to just add the plugin at the suer config level such as 

```
{
    resolve: 'gatsby-plugin-page-creator',
    option: {
        path: './src/pages'
    }
}
```